### DISCUSSION TITLE
add contactform to store details

### DISCUSSION THREAD

**Discussion by elcreativo** [Tastaturberuf/anyStores#100] [Feature Requests / Funktionswünsche]
Wie würdest du ein Kontaktformular auf jede Detailseite unterbringen bei dem die Empfänger Adresse jeweils die Empfängeradresse aus den Store-Details ist?

Mit dem Formulargenerator muss ja vorab schon eine fixe Empfänger Adresse  hinterlegt werden.

**Comment by Tastaturberuf** [comment]
Wahrscheinlich mit EFG und etwas PHP-Logik im Template. Aber eine konkrete Lösung habe ich nicht parat, lässt sicher aber erarbeiten.
